Basic/tutorial-applestamp/contracts/src/main/java/com/tutorial/contracts/AppleStampContract.java

```diff
@@ -41,6 +38,7 @@ public void verify(@NotNull LedgerTransaction tx) throws IllegalArgumentExceptio
     // Used to indicate the transaction's intent.
     public interface Commands extends CommandData {
         //In our hello-world app, We will have two commands.
-        class Issue implements AppleStampContract.Commands {}
+        class Issue implements AppleStampContract.Commands {
```

Comment:
Though it's done by Intellij auto-formatting, `{}` looks good to me too. Please let me know if this shouldn't be changed. I'll revert it.